x-pack/functionbeat/docs/config-options.asciidoc

```diff
@@ -0,0 +1,102 @@
+[id="configuration-{beatname_lc}-options"]
+== Configure functions
+
+++++
+<titleabbrev>Configure functions</titleabbrev>
+++++
+
+{beatname_uc} runs as a function in your serverless environment.
+
+Before deploying {beatname_uc}, you need to configure one or more functions and
+specify details about the services that will trigger the functions.
+
+You configure the functions in the the +{beatname_lc}.yml+ configuration file.
+When you're done, you can <<{beatname_lc}-deploying,deploy the functions>>
+to your serverless environment.
+
+The following example configures two functions: `cloudwatch` and `sqs`. The
+`cloudwatch` function collects events from CloudWatch Logs. The `sqs` function
+collects messages from Amazon Simple Queue Service (SQS). Both functions forward
+the events to {es}.
+
+["source","sh",subs="attributes"]
+----
+{beatname_lc}.provider.aws.deploy_bucket: "functionbeat-deploy"
+{beatname_lc}.provider.aws.functions:
+  - name: cloudwatch
+    enabled: true
+    type: cloudwatch_logs
+    description: "lambda function for cloudwatch logs"
+    triggers:
+      - log_group_name: /aws/lambda/my-lambda-function
+        filter_pattern: mylog_
```

Comment:
I think by default we want all events so I would either comment or remove the _filter_pattern_